app/components/ChartSettings/modules/Metadata.js

```diff
@@ -18,16 +18,28 @@ export default class Metadata extends Component {
     title: '',
     caption: '',
     credit: '',
+    subtitle: false,
   };
 
   componentWillMount() {
     this.setState(this.props.metadata);
+    if ('undefined' !== typeof this.props.metadata.subtitle &&
+      false !== this.props.metadata.subtitle) {
```

Comment:
Switched it over to the check you recommended - smarter to check for the explicit values we'd be expecting rather than any falsy value.